### PR TITLE
Akamai WAF SIEM Integration

### PR DESCRIPTION
**Pull request by GalRabin**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16613

## Description
Akamai WAF SIEM integration by https://developer.akamai.com/api/cloud_security/siem/v1.html

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Additional changes
Pipfile copy from phishlabs drp so freeze-gun will run

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/akamai_siem/Integrations/Akamai_SIEM/Akamai_SIEM.yml)

### CODE REVIEW

**Comment by GalRabin** [comment]
Hi @kirbles19 ,
Can you please review the following:
1. [YML file](https://github.com/demisto/content/blob/akamai_siem/Integrations/Akamai_SIEM/Akamai_SIEM.yml)
2. [Readme file](https://github.com/demisto/content/blob/akamai_siem/Integrations/Akamai_SIEM/README.md)

File: Integrations/Akamai_SIEM/Akamai_SIEM.yml [hunk 1]
```diff
@@ -21,21 +21,21 @@ configuration:
...
+  display: Config IDs to fetch
   name: configIds
   required: true
   type: 0
 - display: Incident type
   name: incidentType
   required: false
   type: 13
-- defaultvalue: 1 hours
+- defaultvalue: 1 hour
```
Comment (GalRabin) [inline Integrations/Akamai_SIEM/Akamai_SIEM.yml]: The functionality requires hours not hour, it could be 2 hours etc

**Comment by kirbles19** [comment]
@GalRabin Done.

File: Integrations/Akamai_SIEM/Akamai_SIEM.yml [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  required: false
+  type: 8
+description: Use the Akamai WAF SIEM integration to retrieve security events from Akamai Web Application Firewall (WAF) service.
+display: Akamai WAF SIEM
+name: Akamai WAF SIEM
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: 'Unique identifier for each security configuration. To report on
```
[the remark this answers is not in the document]
Comment (GalRabin) [reply Integrations/Akamai_SIEM/Akamai_SIEM.yml]: I am using it when querying the API if it necessary I can change it

**Pull request merged**